src/Workspaces/CoreTest/CodeStyle/NamingStylePreferencesUpgradeTests.cs

```diff
@@ -20,15 +20,15 @@ private static void AssertTrimmedEqual(string expected, string actual)
             Assert.Equal(expected.Trim(), actual.Trim());
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.NamingStyle)]
+        [Fact(Skip = "PROTOTYPE(func-ptr): Adding a typekind is changing Struct to it's other value, Structure"), Trait(Traits.Feature, Traits.Features.NamingStyle)]
```

Comment:
>"PROTOTYPE(func-ptr): Adding a typekind is changing Struct to it's other value, Structure" [](start = 21, length = 90)

The number ```FunctionPointer = 13,``` definitely has something to do with this.

Reply:
Yeah. Current guess is that the .NET serializer does a binary search among a sorted list of names, and it doesn't do anything special for duplicate values. Will need to work with the IDE as to whether they care.

---
In reply to: [362996793](https://github.com/dotnet/roslyn/pull/40431#discussion_r362996793) [](ancestors = 362996793)